packages/client/src/runtime/utils/sql-template-tag.ts

```diff
@@ -0,0 +1,16 @@
+// Wrapper script to work around the type bug of sql-template-tag@4.1.0
+
+import { empty, join as joinOriginal, raw, Sql, sqltag as sqltagOriginal } from 'sql-template-tag'
+
+declare function joinFunc(values: RawValue[], separator?: string): Sql
+declare function sqltagFunc(strings: ReadonlyArray<string>, ...values: RawValue[]): Sql
+
+const join = joinOriginal as typeof joinFunc
+const sqltag = sqltagOriginal as typeof sqltagFunc
```

Comment:
This looks good although I prefer the alternative with `generateInFolder`, if you have time and if it actually works out that easily. This seems to be viable, but of course it adds that thin layer between us and the library, which I'd like to avoid if possible. If not, `__tests__` are being phased out, and we can add a TODO as a comment here that when we don't depend on `getTestClient` anymore, we can basically upgrade to v5 since we bundle to CJS.

Reply:
Ok, I'll experiment with `generateInFolder` for a bit.

Reply:
If it is too tricky to get the integration-tests to pass, just leave it as is and ping me.